README.md

```diff
@@ -154,32 +154,21 @@ To set up your environment for development, do the following:
 chmod a+x core/*
 ```
 
-1. Pull in volttron from the [official volttron repo](https://github.com/VOLTTRON/volttron) using the following git command:
+1. Build the image locally:
 
+* Using docker buildx (preferred)
 ```bash
-# Clones https://github.com/VOLTTRON/volttron.git into the 'volttron' directory
-git submodule update --init --recursive
+docker buildx build -f Dockerfile-dev --no-cache --force-rm .
 ```
 
-Why are we doing this? This repo has a directory called 'volttron', which contains the volttron codebase. In other words, this repo contains another repo in a subfolder.
-When you initially clone this repo, the 'volttron' directory is empty. This directory contains the volttron codebase used to create the volttron platform.
+* Dockerfile will clone the volttron repository in the image from https://github.com/VOLTTRON/volttron.git 
+  and defaults to the main branch. Dockerfile-dev defaults to the develop branch. Other than that, they are 
+  identical. If you would like to use a different repository, or branch, you can set the following build 
+  arguments (specified with the --build-arg flag):
+    - volttron_repo: The repository to clone. Use the https URL for the repo.
+    - volttron_git_branch: The branch or tag to clone. Defaults to main. 
```

Comment:
Can you add a full example inline code that has both of these args set in the build?